src/menu/views/credits.c

```diff
@@ -44,6 +44,7 @@ static void draw (menu_t *menu, surface_t *d) {
         " Robin Jones / NetworkFusion\n"
         " Mateusz Faderewski / Polprzewodnikowy\n"
         "Contributors:\n"
+        " MaxiB : Controller pak menu manager\n"
```

Comment:
Controller Pak should be uppercased.

Reply:
I dont want to be a stick in the mud, but it is more than that... Until we have a tab for contributors, this needs to be added to the `Readme.md` Rather than the menu. But, Also, improve the https://github.com/Polprzewodnikowy/N64FlashcartMenu/blob/develop/docs/15_controller_paks.md